common/src/main/java/com/wynntils/core/consumers/commands/CommandManager.java

```diff
@@ -59,60 +67,36 @@ public CommandManager() {
         registerAllCommands();
     }
 
+    @SubscribeEvent
+    public void onCommandsAdded(CommandsAddedEvent event) {
+        for (Command command : commandInstanceSet) {
+            // Wynntils command is special,
+            // it registers every other command as a subcommand
+            if (command instanceof WynntilsCommand wynntilsCommand) {
```

Comment:
Maybe we should store wynntilsCommand as a field when registering it, so you don't have to check for it in the for loop?

Reply:
I mean, if you want to, okay. This runs once per server join, basically not related to performance in any way.

Reply:
If this is the only thing that needs changing, I would merge this as I am off for today..

Reply:
It is not for performance as much as for readability.

Reply:
I had some more comments below. We can merge this tomorrow, no hurry.